ArgosIOS/Argos/Views/LandingPage/LandingView.swift

```diff
@@ -43,7 +43,7 @@ struct LandingView: View {
                     ArgosButton(title: "Map View", action: {
                         self.viewModel.onMapViewClicked()
                     }).navigationDestination(isPresented: self.$viewModel.showMap) {
-                        MapView()
+                        MapView(latitude:self.viewModel.latitude, longitude: self.viewModel.longitude)
```

Comment:
we shouldnt be passing this into the map view, the map views viewmodel should handle this